File: java-maven/plugins/amqp/src/test/java/{{ .PackagePath }}/{{ .NameWord }}/scenarios/AmqpScenario.java
package {{ .Package }}.{{ .NameWord }}.scenarios;

import io.gatling.javaapi.core.ScenarioBuilder;
import {{ .Package }}.{{ .NameWord }}.cases.AmqpActions;

import static io.gatling.javaapi.core.CoreDsl.scenario;

public final class AmqpScenario {

    private AmqpScenario() {
    }

    public static ScenarioBuilder create() {
        return scenario("Amqp Scenario")
                .exec(AmqpActions.publishMessage);
    }
}

File: kotlin-gradle/plugins/amqp/src/gatling/kotlin/{{ .PackagePath }}/{{ .NameWord }}/scenarios/AmqpScenario.kt
package {{ .Package }}.{{ .NameWord }}.scenarios

import io.gatling.javaapi.core.CoreDsl.scenario
import io.gatling.javaapi.core.ScenarioBuilder
import {{ .Package }}.{{ .NameWord }}.cases.AmqpActions

object AmqpScenario {
    fun create(): ScenarioBuilder =
        scenario("Amqp Scenario")
            .exec(AmqpActions.publishMessage)
}
